src/ragapp/backend/routers/chat/vercel_response.py

```diff
@@ -0,0 +1,193 @@
+import json
```

Comment:
can we copy this file to CL and use it here and there?

Reply:
Absolutely. How about save this for the next refactoring PR?

Reply:
yes, please keep it on your list